use/profiles.js

```diff
@@ -15,375 +15,370 @@ export const useProfiles = () => {
 
   const genderList = [
     {
-      label: 'Female',
+      label: t('profiles.genderList.female'),
       value: 'Female',
     },
     {
-      label: 'Male',
+      label: t('profiles.genderList.male'),
       value: 'Male',
     },
     {
-      label: 'Other',
+      label: t('profiles.genderList.female'),
```

Comment:
```suggestion
      label: t('profiles.genderList.other'),
```

Reply:
Right. I think @amanuela97 missed this part. ;)